1/tests/unit/__snapshots__/ListItem.spec.js.snap

```diff
@@ -0,0 +1,93 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ListItem given a \`todo\` Click on \`Edit\` button edit text and submit has the expected html structure 1`] = `
+<li>
+  <div
+    class="itemContainer"
+  >
+    <span
+      class="messageSection"
+    >
+      <span>
+        Moi
+      </span>
+    </span>
+     
+    <div
+      class="mb5"
+    >
+      <span
+        class="buttonContainer"
+      >
+        <a>
+          <img
+            class="icon"
+            src="../assets/edit_circ.png"
+            title="Edit"
+          />
+        </a>
+         
+        <span
+          class="space"
+        />
+         
+        <a>
+          <img
+            class="icon"
+            src="../assets/delete_circ.png"
+            title="Delete"
+          />
+        </a>
+      </span>
+       
+      <!---->
+    </div>
+  </div>
+</li>
+`;
+
+exports[`ListItem given a \`todo\` has the expected html structure 1`] = `
+<li>
+  <div
+    class="itemContainer"
+  >
+    <span
+      class="messageSection"
+    >
+      <span>
+        Foo
+      </span>
+    </span>
+     
+    <div
+      class="mb5"
+    >
+      <span
+        class="buttonContainer"
+      >
+        <a>
+          <img
+            class="icon"
+            src="../assets/edit_circ.png"
+            title="Edit"
+          />
+        </a>
+         
+        <span
+          class="space"
+        />
+         
+        <a>
+          <img
+            class="icon"
+            src="../assets/delete_circ.png"
+            title="Delete"
+          />
+        </a>
+      </span>
+       
+      <!---->
+    </div>
+  </div>
+</li>
+`;
```

Comment:
Is there a way to restrict the snapshot to only the part of the HTML that has changed? I don't know myself because I rarely use snapshots. I assume there is a way though and then it would increase readability if you could focus the reader's attention to the relevant html.

Reply:
Also the test description could be better, for me it's hard to to understand what's the "expected html structure" here.